app/src/main/java/com/grupo3/vinilos/album/list/AlbumsViewModel.kt

```diff
@@ -36,4 +36,17 @@ class AlbumsViewModel : ViewModel() {
         }
 
     }
+
+    suspend fun createAlbum(album: AlbumDto) {
```

Comment:
podemos hacer un wrapper de la funcion en un viewModelScope?
```suggestion
    fun createAlbum(album: AlbumDto) {
    viewModelScope.launch(){
```
Para aplicar las recomendaciones del monitor ?

Reply:
Creo que ya no es necesario debido a que se esta usando el `Dispatcher.IO` para realizar la operación. Siendo así, la vista es la encargada de llamar la función de la siguiente forma

```kotlin
LaunchedEffect(Unit) {
        withContext(Dispatchers.IO) {
            viewModel.createAlbum(...)
        }
    }
```

Reply:
Y como va a ser lanzada desde el boton ?

Reply:
Ah, en ese caso igual se puede llamar al dispatcher y cambiar el estado de la pantalla para que se muestre un loading

Reply:
Aunque revisando los comentarios del monitor, debería de estar tal y como mencionas Chris. Lo voy a agregar

Reply:
Reisando, la función ya se puede encapuslar en un viewModelScope desde la vista. Lo marcaré como resuelto